app/swagger/requests/appointments.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+
+module Swagger
+  module Requests
+    class Appointments
+      include Swagger::Blocks
+
+      swagger_path '/v0/appointments' do
+        operation :get do
+          extend Swagger::Responses::AuthenticationError
+
+          key :description, 'List of user appointments for the previous three months, through the upcoming six months'
+          key :operationId, 'getAppointments'
+          key :tags, %w[
+            appointments
+          ]
+
+          parameter :authorization
+
+          response 200 do
+            key :description, 'Response is OK'
+            schema do
+              key :required, [:data]
+
+              property :data, type: :object do
+                key :required, [:attributes]
+                property :attributes, type: :object do
```

Comment:
I'm not super familiar with swagger syntax... is this showing an array of appointment objects?

Reply:
(^ The description says "List of user appointments" but I don't see an `items`.)

Reply:
Nice catch!